lib/zold/commands/remote.rb

```diff
@@ -89,6 +91,21 @@ def run(args = [])
         o.bool '--reboot',
           'Exit if any node reports version higher than we have',
           default: false
+        # @todo #292:30min Group options by subcommands
+        #  Having all the options in one place _rather than grouping them by subcommands_
+        #  makes the help totally misleading and hard to read.
+        #
```

Comment:
@botanicus I think we should get rid of empty lines here, since PDD tool will not understand this block as one piece. In general, stay as far away from empty lines as you can, they are code smell: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html